CODEOWNERS

```diff
@@ -29,64 +29,74 @@ Jenkinsfile  @openvinotoolkit/openvino-admins
-/src/inference/include/ie/vpu/  @openvinotoolkit/openvino-ie-vpu-maintainers
-/src/tests_deprecated/unit/engines/vpu/  @openvinotoolkit/openvino-ie-vpu-maintainers @openvinotoolkit/openvino-ie-tests-maintainers
-/src/tests_deprecated/functional/vpu/  @openvinotoolkit/openvino-ie-vpu-maintainers @openvinotoolkit/openvino-ie-tests-maintainers
-/src/tests_deprecated/behavior/vpu/  @openvinotoolkit/openvino-ie-vpu-maintainers @openvinotoolkit/openvino-ie-tests-maintainers
-/src/tests/functional/plugin/myriad/  @openvinotoolkit/openvino-ie-vpu-maintainers @openvinotoolkit/openvino-ie-tests-maintainers
-/src/tests/unit/vpu/  @openvinotoolkit/openvino-ie-vpu-maintainers @openvinotoolkit/openvino-ie-tests-maintainers
-/src/tests/unit/engines/vpu/  @openvinotoolkit/openvino-ie-vpu-maintainers @openvinotoolkit/openvino-ie-tests-maintainers
+/src/tests/**myriad** @openvinotoolkit/openvino-ie-vpu-maintainers
 
 # OpenVINO GNA:
 /src/plugins/intel_gna/  @openvinotoolkit/openvino-ie-gna-maintainers
-/src/inference/include/ie/gna/  @openvinotoolkit/openvino-ie-gna-maintainers
+/src/tests/**gna**  @openvinotoolkit/openvino-ie-gna-maintainers
 
 # OpenVINO ARM CPU:
 /docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md  @openvinotoolkit/openvino_contrib-arm_plugin-maintainers
 
 # OpenVINO Auto (MULTI) plugin:
 /src/plugins/auto/  @openvinotoolkit/openvino-ie-auto-multi-maintainers
-/src/inference/include/ie/multi-device/  @openvinotoolkit/openvino-ie-auto-multi-maintainers
 
-# OpenVINO Paddle frontend:
+# OpenVINO Auto (Batch) plugin:
+/src/plugins/auto_batch/ @openvinotoolkit/openvino-auto-batch-maintainers
+
+# OpenVINO Hetero plugin:
+/src/plugins/hetero/ @openvinotoolkit/openvino-hetero-maintainers
+
+
+# OpenVINO Frontends:
+/src/frontends/common/ @openvinotoolkit/openvino-common-frontend-maintainers
+/src/frontends/ir/ @openvinotoolkit/openvino-ir-frontend-maintainers
+/src/frontends/onnx/ @openvinotoolkit/openvino-onnx-frontend-maintainers
+/src/frontends/tests/onnx/ @openvinotoolkit/openvino-onnx-frontend-maintainers
 /src/frontends/paddle/  @openvinotoolkit/openvino-ie-paddle-maintainers
+/src/frontends/tensorflow/  @openvinotoolkit/openvino-tf-frontend-maintainers
 
 # OpenVINO Tests:
 /src/tests/  @openvinotoolkit/openvino-ie-tests-maintainers @openvinotoolkit/openvino-ie-test-developers
 /src/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers @openvinotoolkit/openvino-ie-test-developers
-/src/tests/functional/inference_engine/ngraph_reader/  @openvinotoolkit/openvino-ie-tests-maintainers @openvinotoolkit/openvino-ngraph-maintainers
-/src/tests/functional/inference_engine/transformations/  @openvinotoolkit/openvino-ie-tests-maintainers @openvinotoolkit/openvino-ngraph-maintainers
+/src/frontends/tests/frontend/shared/ @openvinotoolkit/openvino-ie-tests-maintainers
 
 # Documentation:
 /docs/  @openvinotoolkit/openvino-docs-maintainers
-/docs/template_plugin/  @openvinotoolkit/openvino-ie-template-maintainers
```

Comment:
I think we still need to have `/docs/template_plugin/` and also `/docs/template_extension/`

Reply:
Good catch. I'll fix it.